cmd/root.go

```diff
@@ -46,6 +49,18 @@ func NewRootCmd() *cobra.Command {
 		Short: "proxy-init adds a Kubernetes pod to the Linkerd service mesh",
 		Long:  "proxy-init adds a Kubernetes pod to the Linkerd service mesh.",
 		RunE: func(cmd *cobra.Command, args []string) error {
+
+			if options.TimeoutCloseWaitSecs != 0 {
```

Comment:
Should we assert the value here is greater than 0? Or at least: `if options.TimeoutCloseWaitSecs > 0`

Reply:
It wouldn't hurt to be super defensive like that, but hopefully not necessary.  The injector code which sets this flag sets it from a go `time.Duration` which cannot be negative.